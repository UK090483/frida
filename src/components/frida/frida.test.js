import React from "react"
import renderer from "react-test-renderer"
import Frida from "./frida"
import "jest-styled-components"

describe("Frida", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Frida />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders correctly with props", () => {
    const tree = renderer.create(<Frida text={"test"} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
