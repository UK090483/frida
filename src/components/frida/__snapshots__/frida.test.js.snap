// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Frida renders correctly 1`] = `
<span
  className="sc-bdnylx kKa-dPu"
>
  #Meet
  <span
    className="sc-gtssRu fGCCyX"
  >
    Frida
  </span>
</span>
`;

exports[`Frida renders correctly with props 1`] = `
<span
  className="sc-bdnylx kKa-dPu"
>
  #Meet
  <span
    className="sc-gtssRu fGCCyX"
  >
    test
  </span>
</span>
`;
